app/javascript/packs/application.js

```diff
@@ -27,6 +27,8 @@ require('../partials/_leaflet_map');
 // ckeditor custom build
 require('../ckeditor');
 
+require('./fileupload')
```

Comment:
this file better should have not been placed in `packs`, as packs are auto generated files. i would suggest to place it next to `ckeditor` or in `partials`, as this js is used for the `_fileupload` erb partial? what do you think?

Reply:
i will take this into account when working on fixes for the upload requested in SVA-626

Reply:
done with #181